staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -116,7 +116,9 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringSliceVar(&s.EtcdServersOverrides, "etcd-servers-overrides", s.EtcdServersOverrides, ""+
 		"Per-resource etcd servers overrides, comma separated. The individual override "+
-		"format: group/resource#servers, where servers are URLs, semicolon separated.")
+		"format: group/resource#servers, where servers are URLs, semicolon separated. "+
+		"Note that this applies only to static API types compiled into this server binary, "+
+		"so it does not affect aggregated APIs or dynamic CRDs.")
```

Comment:
Note that CRD stands for Custom Resource _Definition_, and the concern here is the storage for the resources rather than their definitions; note that Jordan had one more word at the end of his suggestion.  I myself might phrase it slightly differently, something like "... does not affect aggregated APIs nor ones defined by CRDs".  Or, for greater consistency with the rest of the description of this flag --- which starts out by using the term "resource" --- something like "... does not affect resources served by aggregated apiservers nor ones defined by CRDs".

Reply:
I rephrased the comment as suggested, thanks!